lua/trouble/providers/lsp.lua

```diff
@@ -81,6 +81,27 @@ function M.definitions(win, buf, cb, _options)
   end)
 end
 
+---@return Item[]
+function M.type_definitions(win, buf, cb, _options)
+  local method = "textDocument/typeDefinition"
+  local params = util.make_position_params(win, buf)
+  lsp.buf_request(buf, method, params, function(err, result)
```

Comment:
this should be `lsp_buf_request` instead of `lsp.buf_request`, so we can make it bakward compatible with 0.5

Reply:
missed that, thanks!